komodo-relational-commands/src/main/java/org/komodo/relational/commands/server/WkspStatusServerManager.java

```diff
@@ -55,6 +55,9 @@ public static WkspStatusServerManager getInstance(final WorkspaceStatus wsStatus
      * @throws KException if error occurs
      */
     public static void reset() throws KException {
+        if (instance == null)
```

Comment:
Can we still have the `instance` class variable (which contains a WorkspaceStatus, which contains a transaction, which contains user credentials) now that we've gone multiuser?